src/types/UiaTextRangeBase.cpp

```diff
@@ -601,11 +612,15 @@ IFACEMETHODIMP UiaTextRangeBase::Move(_In_ TextUnit unit,
     _outputRowConversions();
 #endif
 
-    auto moveFunc = &_moveByDocument;
+    std::function<std::pair<Endpoint, Endpoint>(gsl::not_null<IUiaData*>, const int, const MoveState, gsl::not_null<int*> const)> moveFunc = &_moveByDocument;

```

Comment:
That's a mouthful. Maybe we should typedecl this?

Reply:
`decltype`d it. Reduced the extra code, but still the same result.